### PR TITLE
Feat/webapp/new login

### PR DESCRIPTION
**Pull request by gony02**
Creating the login view with the corresponding tests and internationalisation. Two of the tests cannot pass.

### CODE REVIEW

**Review by sergiorodriguezgarcia — APPROVED** [review]
All seems to be correct.

**Comment by jjgancfer** [comment]
I'm currently waiting for the bug in #69 to be solved, as I cannot work towards fixing the test that fails without it. I'll approve when it is.

File: webapp/src/pages/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            <Stack flexDir={"column"} mb="2" justifyContent="center" alignItems={"center"}>
+                <Avatar bg="blue.500" />
+                <Heading as="h2" color="blue.400">{ t("common.login")}</Heading>
+                { 
+                    !hasError ? 
+                    <></> : 
+                    <Center bgColor={"#FFA98A"} margin={"1vh 0vw"} padding={"1vh 0vw"} 
+                        color={"#FF0500"} border={"0.1875em solid #FF0500"}
+                        borderRadius={"0.75em"} maxW={"100%"} minW={"30%"}>
+                            <Text>Error</Text>
```
Comment (jjgancfer) [inline webapp/src/pages/Login.jsx:45]: This error message does not seem to be localized. It is also quite generic, so please change it.

**Review by jjgancfer — CHANGES_REQUESTED** [review]

**Review by jjgancfer — APPROVED** [review]
The requested changes seem to have been included. Approved.

**Pull request merged**